move-mutation-test/src/mutation_test.rs

```diff
@@ -0,0 +1,152 @@
+            experiments: experiments_from_opt_level(&cfg.move_pkg.optimize),
+        },
+        ..Default::default()
+    };
+
+    let natives = aptos_debug_natives(NativeGasParameters::zeros(), MiscGasParameters::zeros());
+    let cost_table = None;
+    let gas_limit = None; // unlimited.
+                          // TODO(M2): Add special handling for the coverage computation.
+    let compute_coverage = false;
+
+    let result = move_cli::base::test::run_move_unit_tests(
+        package_path,
+        config.clone(),
+        UnitTestingConfig {
+            filter: cfg.filter.clone(),
+            report_stacktrace_on_abort: true,
+            report_storage_on_error: cfg.dump_state,
+            ignore_compile_warnings: cfg.ignore_compile_warnings,
+            named_address_values: cfg
+                .move_pkg
+                .named_addresses()
+                .iter()
+                .map(|(name, account_address)| {
+                    (
+                        name.clone(),
+                        NumericalAddress::from_account_address(*account_address),
+                    )
+                })
+                .collect(),
+            ..UnitTestingConfig::default()
+        },
+        natives,
+        aptos_test_feature_flags_genesis(),
+        gas_limit,
+        cost_table,
+        compute_coverage,
+        &mut error_writer,
+    )
+    .map_err(|err| Error::msg(format!("failed to run unit tests: {err:#}")))?;
+
+    match result {
+        UnitTestResult::Success => Ok(()),
+        UnitTestResult::Failure => Err(Error::msg("Move unit test error")),
+    }
+}
+
+/// Get debug natives.
+// move_stdlib has the testing feature enabled to include debug native functions
+fn aptos_debug_natives(
+    native_gas_parameters: NativeGasParameters,
+    misc_gas_params: MiscGasParameters,
+) -> NativeFunctionTable {
+    // As a side effect, also configure for unit testing
+    natives::configure_for_unit_test();
+    extended_checks::configure_extended_checks_for_unit_test();
+    // Return all natives -- build with the 'testing' feature, therefore containing
+    // debug related functions.
+    natives::aptos_natives(
+        LATEST_GAS_FEATURE_VERSION,
+        native_gas_parameters,
+        misc_gas_params,
+        TimedFeaturesBuilder::enable_all().build(),
+        Features::default(),
+    )
+}
+
+/// Get bytecode version.
+fn get_bytecode_version(
+    bytecode_version_in: Option<u32>,
+    language_version: Option<LanguageVersion>,
+) -> Option<u32> {
+    if bytecode_version_in.is_none() {
+        if let Some(language_version) = language_version {
+            Some(language_version.infer_bytecode_version(bytecode_version_in))
+        } else {
+            bytecode_version_in
+        }
+    } else {
+        bytecode_version_in
+    }
+}
+
+/// Get a list of stringified [`Experiment`] from the optimization level.
+fn experiments_from_opt_level(optlevel: &Option<OptimizationLevel>) -> Vec<String> {
```

Comment:
Nope, unfortunately, the EXPERIMENT part won't work - it's a struct.

```rust
use OptimizationLevel::*;
```
This does work, but then we have to specify `std::Option::None` and `std::Option::Some` explicitly since we have a collision between the same enum names.
So it's best to keep it this way.